src/Api/Vault/AuthorizationHandlers/Collections/CollectionAuthorizationHandler.cs

```diff
@@ -29,7 +31,7 @@ public class CollectionAuthorizationHandler : BulkAuthorizationHandler<Collectio
     }
 
     protected override async Task HandleRequirementAsync(AuthorizationHandlerContext context,
-        CollectionOperationRequirement requirement, ICollection<Collection> resources)
+        CollectionOperationRequirement requirement, ICollection<Collection>? resources)
```

Comment:
Do we need to add the nullable modifier to `ICollection<Collection>`

Reply:
We decided (at some point, can't find a link at the moment) to let the auth handlers be responsible for checking for a null resource. So, once I added `#nullable enable`, I started getting warnings about the `resource == null` check below always being false according to the nullable annotations.

Admittedly, this check is slightly redundant here because this handler extends the `BulkAuthorizationHandler` which already does null checking for us. With that said, I still feel its beneficial to continue being explicit with the null check, even if only to continue setting a precedent for future auth handlers that may not use `BulkAuthorizationHandler`. If you think otherwise let me know!

Reply:
That makes sense to me! Thanks for taking the time to explain it